android/src/main/kotlin/com/hypertrack/sdk/flutter/common/Result.kt

```diff
@@ -0,0 +1,37 @@
+package com.hypertrack.sdk.flutter.common
+
+internal sealed class Result<S> {
+    fun <T> flatMap(function: (S) -> Result<T>): Result<T> {
+        return when (this) {
+            is Success -> {
+                function.invoke(this.success)
+            }
+            is Failure -> {
+                Failure<T>(this.failure)
+            }
+        }
+    }
+
+    fun forceUnwrap(): S {
```

Comment:
It's not force unwrapping, but it's more a `get()` function, so maybe `getOrThrow()` is a better name?
Unfortunately, from what I see, `!!` operator can't be overloaded...

Reply:
fixed